packages/react-textarea/Spec.md

```diff
@@ -2,62 +2,174 @@
+| [React Bootstrap](https://react-bootstrap.github.io/forms/input-group/)          | `FormControl` as `textarea`     |
+| [Blueprint](https://blueprintjs.com/docs/#core/components/text-inputs.text-area) | `TextArea`                      |
+| [Grommet](https://v2.grommet.io/textarea)                                        | `TextArea`                      |
+
+- Most libraries have their own `TextArea` component while others make it a variant of another component.
+  - `React Bootstrap` uses as with `FormControl` as follows: `<FormControl as="textarea" />`
+  - `Material UI` and `v8` use the `TextField` control and added a variant `multiline` as follows: `<TextField multiline defaultValue="Default Value" />`
+  - `Ant Design` uses the `TextArea` subcomponent as follows: `<Input.TextArea placeholder="Text Area" />`
+
+#### TextArea in v8
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+- `Textfield` component with `multiline` property. When the `multiline` property is set to true, it is rendered as a `textarea`.
+- Has descriptions, label, error message, and handles maximum character count.
+
+#### TextArea in v0
+
+- Doesn't have `label` prop.
+- Handles maximum character count.
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+```tsx
+<TextArea
+  className="rootClassName"
+  style={{ background: 'blue' }}
+  id="textarea-1"
+  defaultValue="This is a TextArea"
+  onChange={(ev, data) => console.log(data.value)}
+  textArea={{ className: 'textAreaClassName', style: { color: 'green' } }}
+/>
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+- TextArea has six variants:
+  - `Filled Darker`: TextArea has a gray background and no outline.
+  - `Filled Lighter`: TextArea has a white background and no outline.
+  - `Underline`: TextArea has a gray background and a line on the bottom.
+  - `Outline`: TextArea has a white background, outline, and a line on the bottom.
+  - `Disabled`: TextArea has a gray background, gray text, outline, and interaction is disabled.
+  - `Read Only`: TextArea has a gray background, gray outline, and typing is disabled.
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+### Component props:
+
+```ts
+type TextAreaProps = {
+  /**
+   * Display inline instead of block.
+   *
+   * @defaultvalue false
+   */
+  inline?: boolean;
+
+  /**
+   * Which direction the TextArea is allowed to be resized.
+   *
+   * @defaultvalue off
+   */
+  resize?: 'off' | 'horizontal' | 'vertical' | 'both';
+
+  /**
+   * Direction the TextArea should resize.
+   *
+   * @defaultvalue off
+   */
+  autoResize?: 'off' | 'top' | 'bottom';
```

Comment:
This turns out to be really tricky to implement properly (there are known bugs with the v8 version), so I'd prefer to leave it out of the initial implementation if possible.

That's a general principle we've been trying to follow for new components: if a feature is hard to implement well and not immediately necessary to product scenarios, leave it out of the initial version.

Reply:
Sounds good, I'll remove it and add a note about it.